daemon/web/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './components/App';
+import InertiaClient from './client';
+
+// Define where the Inertia daemon is hosted.
+const daemonAddress = process.env.NODE_ENV === 'production'
+  ? process.env.DAEMON_ADDRESS
```

Comment:
How does this variable get set on the server?

Reply:
Plan was to deliver it over SSH when the user runs `inertia $DEPLOYMENT up`, since the address is stored in configuration, and there no point to the admin panel being accessible if no project is `up` yet. 🤔 Thoughts?

Wasn't going to add it to this PR, but I guess I might as well 🤷‍♂️

Reply:
Environments are funny - `inertia` should make them easy 😄We should have a uniform way of setting all of the env vars on the server (whether the user needs to set one or the daemon does).

Side note: I _really_ like being able to set my env variables in heroku/elastic beanstalk. The Inertia web client is a nice place to do this (as long as we're all HTTPSed up 👍).

Reply:
Something to consider usability-wise for #89 💯